specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/customRecommedations.json

```diff
@@ -0,0 +1,384 @@
+            "$ref": "./examples/CustomRecommendations/ListBySecurityConnectorCustomRecommendations_example.json"
+          },
+          "List custom recommendations by management group scope": {
+            "$ref": "./examples/CustomRecommendations/ListByManagementGroupCustomRecommendations_example.json"
+          }
+        },
+        "tags": [
+          "CustomRecommendations"
+        ],
+        "description": "Get a list of all relevant custom recommendations over a scope",
+        "operationId": "CustomRecommendations_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CustomRecommendationsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/customRecommendations/{customRecommendationName}": {
+      "get": {
+        "x-ms-examples": {
+          "Get a custom recommendation over subscription scope": {
+            "$ref": "./examples/CustomRecommendations/GetBySubscriptionCustomRecommendation_example.json"
+          },
+          "Get a custom recommendation over security connector scope": {
+            "$ref": "./examples/CustomRecommendations/GetBySecurityConnectorCustomRecommendation_example.json"
+          },
+          "Get a custom recommendation over management group scope": {
+            "$ref": "./examples/CustomRecommendations/GetByManagementGroupCustomRecommendation_example.json"
+          }
+        },
+        "tags": [
+          "CustomRecommendations"
+        ],
+        "description": "Get a specific custom recommendation for the requested scope by customRecommendationName",
+        "operationId": "CustomRecommendations_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/customRecommendationName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CustomRecommendation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
```

Comment:
>"../../../common/v2/types.json#/definitions/CloudError" [](http://example.com/codeflow?start=22&length=55)

please use common type in ../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse

Reply:
fixed